test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs

```diff
@@ -57,10 +57,8 @@ public Task ResponseCompression_IISExpress_NoCompression(RuntimeFlavor runtimeFl
         [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
         [InlineData(RuntimeFlavor.CoreClr, ApplicationType.Portable, HostingModel.OutOfProcess, "/p:ANCMVersion=V1")]
         [InlineData(RuntimeFlavor.CoreClr, ApplicationType.Portable, HostingModel.OutOfProcess, "/p:ANCMVersion=V2")]
```

Comment:
Shouldn't you also pull all of the v2 tests?

Reply:
Why?

Reply:
V2 isn't being shipped, so no need for the tests?

Reply:
It's still being built, and tests work, why remove?

Reply:
I'm fine with keeping it in.